src/navbar/navbar-items.tsx

```diff
@@ -97,56 +97,67 @@ export const NavbarItems = <T,>({
     // =============================================================================
     const renderItems = () => {
         return items.map((item, index) => {
-            const selected = checkSelected(item);
-            const { children, options, ...otherItemAttrs } = item;
+            switch (item.itemType) {
+                case "component": {
+                    const component = (item && item.children) || null;
+                    return <div key={index}>{component}</div>;
```

Comment:
this should be a list item as it's part of a list

```suggestion
                    return <li key={index}>{component}</li>;
```